package.json

```diff
@@ -183,6 +183,7 @@
     "karma-webpack": "2.0.3",
     "mocha": "3.2.0",
     "ngc-webpack": "1.2.0",
+    "ng2-bootstrap": "1.6.6",
```

Comment:
Please don't add this to the devDep it is already a primary dependency on line 119. And you can't upgrade the version. It will break things if you do.